docs/platforms/platforms_index.rst

```diff
@@ -213,7 +213,7 @@ libEnsemble on specific HPC systems.
     frontier
     perlmutter
     polaris
-    spock/crusher <spock_crusher>
+    Spock/Crusher <spock_crusher>
```

Comment:
Why is this capitalized (while not the others)?

Reply:
Otherwise the system names appear lowercased: https://libensemble.readthedocs.io/en/main/platforms/platforms_index.html#instructions-for-specific-platforms

But since `spock_crusher.rst` has the properly titled names, listing that file in the toctree without overwriting its title should be fine. Just tested locally. I'll make the simple change